parkhangs-webapp/src/components/EventForm.css

```diff
@@ -54,9 +51,37 @@ textarea {
     display: flex;
     flex-direction: column;
     align-items: center;
+    padding: 1rem;
 }
 
-.calendar-date-select{
-    display: flex;
-    flex-direction: column;
-}
```

Comment:
newline please

Reply:
What do you mean

Reply:
nvm I saw the warning on the pr so I thought there wasn't a line break